react-toggle-switch/my-app/src/App.tsx

```diff
@@ -0,0 +1,25 @@
+import './App.css';
+import './ToggleSwitch.css';
+import { ToggleSwitch } from './ToggleSwitch';
+import { useState } from 'react';
+
+export default function App() {
+  const [isOn, setIsOn] = useState(false);
+
+  const handleClick = () => setIsOn((prev) => !prev);
+
+  const className = `toggle-switch${isOn ? ' is-on' : ''}`;
+
+  return (
+    <>
+      <div className={className}>
```

Comment:
These layout divs should go in ToggleSwitch, also. That makes the ToggleSwitch more self-contained and easier to reuse.